tests/test_fieldset.py

```diff
@@ -469,6 +469,7 @@ def test_fieldset_defer_loading_function(zdim, scale_fac, tmpdir, filename='test
     def compute(fieldset):
         # Calculating vertical weighted average
         for f in [fieldset.U, fieldset.V]:
+            f.data = np.array(f.data)
```

Comment:
why is it needed to convert `f.data` to numpy array here? Is that always needed in compute functions? Should we make that clear somewhere?